### PR TITLE
Backslash lint `gtest.h`

### PR DESCRIPTION
**Pull request by Garz4**
Lint backslashes in file `googletest/include/gtest/gtest.h` to have a white space before them.

### CODE REVIEW

**Review by joshiayush — APPROVED** [review]

**Review by Adams214565 — APPROVED** [review]

**Comment by Garz4** [comment]
@asoffer What do you think?

**Review by Adams214565 — APPROVED** [review]

**Comment by Garz4** [comment]
@Adams214565 @joshiayush Can any of you folks approve running workflows? I am a first-time contributor :)

**Review by chacha710 — APPROVED** [review]

**Comment by joshiayush** [comment]
> @Adams214565 @joshiayush Can any of you folks approve running workflows? I am a first-time contributor :)

I'm just like you, an another contributor I can't approve the workflows, sorry :). How about contributing to https://github.com/joshiayush/xtest, it's my project, a mini implementation of googletest ;).

**Comment by asoffer** [comment]
My apologies for being incredibly non-responsive. I'm sure that must have been deeply frustrating, and I don't have an excuse. I just need to get better at being communicative.

To that end, the GoogleTest maintainers and I feel pretty strongly about not manually editing whitespace (in this or any project). Our typical solution, and the one we've employed here in https://github.com/google/googletest/commit/b007c54f2944e193ac44fba1bc997cb65826a0b9, is to use clang-format.

We really appreciate the help; even though we're not accepting this particular PR, it directly led to running clang-tidy and the change linked above. I hope you consider that as much of a contribution as we do, and I hope you feel empowered to continue contributing.

**Comment by Garz4** [comment]
You don't need to apologize, it's ok :) I'm actually grateful this PR led to running clang-tidy, I'm going to keep looking if I can keep contributing :) thanks!

**Pull request closed without merging**
